src/main/scala/io/iohk/ethereum/vm/EvmConfig.scala

```diff
@@ -0,0 +1,156 @@
+package io.iohk.ethereum.vm
+
+// scalastyle:off number.of.methods
+// scalastyle:off magic.number
+object EvmConfig {
+
+  /**
+    * returns the evm config that should be used for given block
+    */
+  def forBlock(blockNumber: BigInt): EvmConfig = {
+    // highest transition block that is less/equal to `blockNumber`
+    transitionBlockToConfigMapping
+      .filterKeys(_ <= blockNumber)
+      .maxBy(_._1)
+      ._2
+  }
+
+  val FrontierConfig = EvmConfig(
+    feeSchedule = FeeSchedule.FrontierFeeSchedule,
+    opCodes = OpCodes.FrontierOpCodes)
+
+  // TODO: exceptional_failed_code_deposit
+  val HomesteadConfig = EvmConfig(
+    feeSchedule = FeeSchedule.HomesteadFeeSchedule,
+    opCodes = OpCodes.HomesteadOpCodes)
+
+  // TODO: sub_gas_cap_divisor, no_empty, kill_empty from schedule.rs, parity
+  val PostEIP150Config = EvmConfig(
+    feeSchedule = FeeSchedule.PostEIP150FeeSchedule,
+    opCodes = OpCodes.PostEIP150OpCodes)
+
+  val frontierBlockNumber = BigInt(0)
+  val homesteadBlockNumber = BigInt(0)
```

Comment:
I think ultimately we should obtain these values from config files (which may be different for real/test/private networks).